apps/marsha/templates/app/job_collectstatic.yml.j2

```diff
@@ -41,7 +41,11 @@ spec:
         # We point to a local registry image build for this "live" image (see
         # ImageStream and BuildConfig templates)
         image: "{{ internal_docker_registry }}/{{ project_name }}/marsha:{{ marsha_image_tag }}-live"
-        command: ["python", "manage.py", "collectstatic", "--noinput"]
```

Comment:
why not just:
```suggestion
command: ["python", "sandbox/manage.py", "collectstatic", "--noinput"]
```

Reply:
This is easier to avoid having to modify the PYTHONPATH. But I made a mistake by modifying Masha's jobs instead of Richie's ones. :sweat_smile: